src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -144,4 +144,17 @@
      * @return The connection
      */
     PlayerConnection getConnection();
+
+    /**
+     * Kicks the player.
```

Comment:
@Aaron1011 what would you suggest as the default behavior, then? `You were kicked`? Or maybe the javadocs should say that the behavior is undefined.